platform_events/test/test.js

```diff
@@ -16,14 +16,502 @@
 
 const assert = require('chai').assert;
 const index = require('../index');
+const awsContext = require('aws-lambda-mock-context');
+const logger = require('../components/logger.js');
+const sinon = require('sinon');
+const AWS = require('aws-sdk-mock');
 
-describe('Sample', function() {
-    it('tests handler', function(done) {
+var event, context, callback, err, errMessage, errType, logMessage, dataObj, callbackObj, logStub;
 
-    	index.handler({method:'GET'},{},function() {})
+describe('platform_events', function() {
 
-        //Test cases to be added here.
-        assert(true);
-        done();
+    var dynamoCheck = function(dynamoMethod, sinonSpy){
+        var serviceName;
+        //assign the correct aws service depending on the method to be used
+        if(dynamoMethod == "scan" || dynamoMethod == "getItem"){
+          serviceName = "DynamoDB";
+        }
+        //mocking DocumentClient from DynamoDB and wrapping with predefined spy
+        AWS.mock(serviceName, dynamoMethod, sinonSpy);
+        //trigger the spy by calling handler()
+        var callFunction = index.handler(event, context, callback);
+        AWS.restore(serviceName);
+        var bool = sinonSpy.called;
+        return bool;
+    };
+
+    beforeEach(function(){
+        spy = sinon.spy();
+        event = {
+            "method" : "",
+            "stage" : "test",
```

Comment:
Most of the tests are failing as you don't have a respective config file for stage "test". Curious, did it pass for you locally?

Reply:
Included test-config.json to resolve this issue